python/looker_sdk/looker-sample.ini

```diff
@@ -1,11 +1,11 @@
 [Looker]
 # API version is required
-api_version=3.1
+api_version=4.0
```

Comment:
I don't think this is required any more?

Reply:
yeah, I missed the gitignore files. will make a pass through those